### PR TITLE
Add simulation lifecycle flowcharts

### PR DESCRIPTION
**Pull request by Felixmil**
related to #1344

### CODE REVIEW

File: vignettes/images/parallel_simulation.drawio.png
Comment (Yuri05) [inline vignettes/images/parallel_simulation.drawio.png:1]: In your original proposal you had "parallel execution" label around the parallel blocks. I found it made the whole picture more clear. Would suggest to add it again (also in the batch simulations picture)
grafik

File: vignettes/images/simulation_lifecycle_flowcharts.drawio
Comment (Yuri05) [inline vignettes/images/simulation_lifecycle_flowcharts.drawio:1]: I assume this file is required only for the png generation, but not once the package is built?
Then let's add it to Rbuildignore

Comment (Felixmil) [reply vignettes/images/simulation_lifecycle_flowcharts.drawio:1]: Done in last commit

File: vignettes/images/parallel_simulation.drawio.png
Comment (Felixmil) [reply vignettes/images/parallel_simulation.drawio.png:1]: Thanks for the review. Added in last commit

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**